tox.ini

```diff
@@ -126,6 +127,7 @@ deps =
     enum34>1.1.0
     azure-storage<=0.36
     prometheus-client==0.5.0
+    tenacity>=6.3.0<7    
```

Comment:
```suggestion
    tenacity>=6.3.0,<7
```

Reply:
Is this change needed? Shouldn't be this pulled in by `install_requires` in `setup.py`?

Reply:
This is unnecessary. Fixed!